src/khoj/database/adapters/__init__.py

```diff
@@ -435,10 +440,14 @@ async def aget_conversation_by_user(
     ):
         if conversation_id:
             conversation = Conversation.objects.filter(user=user, client=client_application, id=conversation_id)
-        else:
+        elif slug:
             conversation = Conversation.objects.filter(user=user, client=client_application, slug=slug)
+        else:
+            conversation = Conversation.objects.filter(user=user, client=client_application)
+
         if await conversation.aexists():
             return await conversation.afirst()
```

Comment:
Should we ensure that we return the most recently updated conversation here as well (like we're doing in `get_converation_by_user` method above)?
```suggestion
        if await conversation.aexists():
            return await conversation.order_by("-updated_at").afirst()
```

Reply:
Updated to add sorting in the filtering step